service/src/main/resources/templates/datadictionary.html

```diff
@@ -0,0 +1,70 @@
+<html xmlns:th="http:/>/www.thymeleaf.org">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<title>DATAWAVE - Data Dictionary</title>
+	<!--/*
+    Loads jQuery, DataTables, some CSS elements for DataTables, and executes `.dataTable()` on the HTML table in the payload.
+    Pagination on the table is turned off, we do an ascending sort on the 1st column (field name) (this is done by default), 
+	and a cookie is saved in the browser that will leave the last sort in place upon revisit of the page.
+	*/-->
+	<link rel="stylesheet" type="text/css" href="/dictionary/css/screen.css" media="screen">
+	<script type="text/javascript" th:src="${jqueryUri}"></script>
+	<script type="text/javascript" th:src="${datatablesUri}"></script>
+	<script type="text/javascript">
+		$(document).ready(function() { 
+			$('#myTable').dataTable({
+				"bPaginate": false, 
+				"bStateSave": true
+			});
+			setTimeout( function() { 
+				$('#myTable').find("td").css("word-break", "break-word"); 
+			}, 4000); 
```

Comment:
Ha! You removed this in the other PR, right?

Reply:
Haha yes! I planned on changing it for this PR too but guess I forgot to go back and delete it. Seems to be still working as expected when I delete it so I'm just going to remove it. I wonder why there was this 4 second wait time to begin with